### PR TITLE
Removed legacy usage of the "tag" field

### PR DESCRIPTION
**Pull request by annando**
There had been some calls for the old "tag" field that now doesn't exist anymore.

### CODE REVIEW

File: src/Model/Tag.php [hunk 1]
```diff
@@ -339,6 +339,23 @@ public static function getByURIId(int $uri_id, array $type = [self::HASHTAG, sel
 		return DBA::selectToArray('tag-view', ['type', 'name', 'url'], $condition);
 	}
 
+	/**
+	 * Return a string with all tags and mentions
+	 *
+	 * @param integer $uri_id
+	 * @return string tags and mentions
+	 */
+	public static function tagTextFromItemId(int $uri_id)
```
Comment (MrPetovan) [inline src/Model/Tag.php:348]: Please rename for clarity:
```suggestion
	public static function getCommaSeparatedStringByURIId(int $uri_id)
```

Comment (annando) [reply src/Model/Tag.php:348]: You could possibly have called it `getACommaSeparatedStringOfHashtagsAndMentionsLikeTheOldFunctionByURIId` instead as well :-)

P.S.: I will apply the changes

Comment (MrPetovan) [reply src/Model/Tag.php]: More words is better than less!

Comment (annando) [reply src/Model/Tag.php]: We are having these discussions at work as well. :-)

**Pull request merged**